src/app/bug/shared/bug.service.spec.ts

```diff
@@ -51,20 +51,42 @@ describe('BugService', () => {
     spectator = createService();
     service = spectator.service;
     
-    mockDataService = spectator.inject(BugDataService);
+    mockDataService = spectator.inject(BugDataService); // incorrect, Hannah says it might not work
   });
 
   it('should be created', () => {
     expect(spectator.service).toBeTruthy();
   });
 
-  describe('getBugsData', () => {
+  /**
+   *  Testing getBugsData() method
+   */
+  describe('getBugsData()', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8891152)
```suggestion
  describe("getBugsData()", () => {
```